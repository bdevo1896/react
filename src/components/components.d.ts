type BaseCommponent = {
  id?: string;
  className?: string;
  style?: React.CSSProperties;
};
