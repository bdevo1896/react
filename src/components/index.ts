export {default as Button} from './Button';
export {default as SVGImage} from './SVGImage';
