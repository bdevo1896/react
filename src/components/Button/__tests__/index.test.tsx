import React from 'react';
import {mount} from 'enzyme';
import {Button} from 'components';

describe('Button', () => {
  it('should render correctly', () => {
    const wrapper = mount(
      <Button onClick={() => console.log('Hey')} type="primary">
        Hey
      </Button>,
    );
    expect(() => wrapper.unmount()).not.toThrow();
    expect(
      <Button onClick={() => console.log('Hey')} type="primary">
        Hey
      </Button>,
    ).toMatchSnapshot();
  });
});
