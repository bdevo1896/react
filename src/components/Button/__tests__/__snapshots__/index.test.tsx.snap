// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render correctly 1`] = `
<Button
  onClick={[Function]}
  type="primary"
>
  Hey
</Button>
`;
